dot/network/status.go

```diff
@@ -123,13 +123,13 @@ func (status *status) validMessage(msg *StatusMessage) bool {
 
 	switch {
 	case msg.GenesisHash != status.hostMessage.GenesisHash:
-		log.Error("Failed to validate status message", "err", "genesis hash")
+		log.Error("[network] Failed to validate status message", "error", "genesis hash")
 		return false
 	case msg.ProtocolVersion < status.hostMessage.MinSupportedVersion:
-		log.Error("Failed to validate status message", "err", "protocol version")
+		log.Error("[network] Failed to validate status message", "error", "protocol version")
 		return false
 	case msg.MinSupportedVersion > status.hostMessage.ProtocolVersion:
-		log.Error("Failed to validate status message", "err", "protocol version")
+		log.Error("[network] Failed to validate status message", "error", "protocol version")
```

Comment:
Shouldn't there be a different error message for each condition?

Reply:
yea, you're probably right, will fix

Reply:
agree, these should be more descriptive like `protocol version mismatch` or something